src/Widget/FormBuilder.php

```diff
@@ -0,0 +1,385 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\Widget;
+
+use Yiisoft\Factory\Exceptions\InvalidConfigException;
+use Yiisoft\Form\FormInterface;
+use Yiisoft\Html\Html;
+use Yiisoft\Http\Method;
+use Yiisoft\Widget\Widget;
+
+final class FormBuilder extends Widget
+{
+    use Collection\Options;
+
+    public const VALIDATION_STATE_ON_CONTAINER = 'container';
+    public const VALIDATION_STATE_ON_INPUT = 'input';
+    private string $action = '';
+    private string $method = Method::POST;
+    private bool $encodeErrorSummary = true;
+    private string $errorCssClass = 'has-error';
+    private string $errorSummaryCssClass = 'error-summary';
+    private string $inputCssClass = 'form-control';
+    private string $requiredCssClass = 'required';
+    private string $successCssClass = 'has-success';
+    private string $validatingCssClass = 'validating';
+    private string $validationStateOn = self::VALIDATION_STATE_ON_CONTAINER;
+    private ?string $validationUrl = null;
+    private bool $validateOnSubmit = true;
```

Comment:
We don't have default JS for form validation so I see no point having these options.

Reply:
Done.